datasets/poleval2019_mt/poleval2019_mt.py

```diff
@@ -0,0 +1,206 @@
+
+    You can then specify the path to this folder for the data_dir argument in the
+    datasets.load_dataset(...) option.
+
+    The <path/to/folder> can e.g. be "/Downloads/English-To-Polish".
+    The data can then be loaded using the following command `datasets.load_dataset("poleval2019_mt", name="en-pl", data_dir="/Downloads/English-To-Polish")`.
+    Name can also be set as "pl-en" when Polish to English dataset is required.
+
+    For Russian to Polish (or vice versa) the link to download the folder is https://drive.google.com/drive/folders/14bpulmsyaou5RPMeGW_u5zmHQamcNQxU
+    Follow the same as above but change the name to "ru-pl" or "pl-ru".
+
+    NOTE: A test set will be automatically downloaded. However it is unavailable for Polish to English task (Not part of Poleval 2019 tasks).
+    """
+
+    BUILDER_CONFIGS = [PolevalMTConfig(language_pair=(key, "pl")) for key, val in _SUPPORTED_LANGUAGES.items()] + [
+        PolevalMTConfig(language_pair=("pl", key)) for key, val in _SUPPORTED_LANGUAGES.items()
+    ]
+
+    def _info(self):
+        source, target = self.config.language_pair
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {"translation": datasets.features.Translation(languages=self.config.language_pair)}
+            ),
+            supervised_keys=(source, target),
+            homepage=_HOMEPAGE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+
+        path_to_manual_file = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
+        if not os.path.exists(path_to_manual_file):
+            raise FileNotFoundError(
+                "{} does not exist. Make sure you insert a manual dir via `datasets.load_dataset('poleval2019_mt', name= .., data_dir=...)` where name can be {} Manual download instructions: {})".format(
+                    path_to_manual_file, "en-pl/pl-en/pl-ru/ru-pl", self.manual_download_instructions
+                )
+            )
+
+        source, target = self.config.language_pair
+
+        # Train & Dev and test path templates
+        path_tmpl = "{dl_dir}/{split}.{lang}"
+        test_tmpl = "{dl_dir}/task4_test/tst_to_{target}.{source}"  # Hardcode alert
+
+        files = {}
+
+        for split in ("train", "dev"):
+            files[split] = {
+                "source_file": path_tmpl.format(dl_dir=path_to_manual_file, split=split, lang=source),
+                "target_file": path_tmpl.format(dl_dir=path_to_manual_file, split=split, lang=target),
+                "split": split,
+            }
+
+        # To handle test split when english is the target language.
+        # This is because there is no Polish to English test file that is available in the default set
+        if "en" == source:
+            dl_dir_test = dl_manager.download_and_extract(_TEST_URL)
+            test_file = "{dl_dir}/task4_test/tst.en".format(dl_dir=dl_dir_test)
```

Comment:
Could you use `os.path.join` here ?
Concatenate paths using `/` can cause issues on windows